Bazel error when running `make images`

I'm not too familiar with bazel so I might be missing something obvious but when trying to build cert-manager on a clean debian:buster image I'm getting a `key "diff_id" not found in dictionary` error. Same problem on MacOSX. I would greatly appreciate any help with this.

```
root@d0633c9f99ae:~/cert-manager# git --no-pager log -1
commit b893ee4e77f03e3a25782f7dba9572541cc96235 (HEAD -> master, origin/release-0.15, origin/master, origin/HEAD)
Merge: c2da21edd 30aace0d5
Author: jetstack-bot <32282838+jetstack-bot@users.noreply.github.com>
Date:   Mon May 4 15:17:48 2020 +0100

    Merge pull request #2880 from meyskens/go1141

    Bump Go to 1.14.2
```

```
root@d0633c9f99ae:~/cert-manager# bazel version
Build label: 3.1.0
Build target: bazel-out/k8-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Wed Apr 22 10:32:27 2020 (1587551547)
Build timestamp: 1587551547
Build timestamp as int: 1587551547
```

```
root@d0633c9f99ae:~/cert-manager# make images
Makefile:104: warning: overriding recipe for target 'ctl'
Makefile:86: warning: ignoring old recipe for target 'ctl'
APP_VERSION= \
DOCKER_REGISTRY= \
bazel run \
	--platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 \
	//build:server-images
ERROR: /root/cert-manager/build/BUILD.bazel:48:1: in container_bundle_ rule //build:push-server-images-amd64:
Traceback (most recent call last):
	File "/root/cert-manager/build/BUILD.bazel", line 48
		container_bundle_(name = 'push-server-images-amd64')
	File "/root/.cache/bazel/_bazel_root/e879c69dfc0f604039abdfad90e59fb0/external/io_bazel_rules_docker/container/bundle.bzl", line 64, in _container_bundle_impl
		_incr_load(ctx, images, <2 more arguments>)
	File "/root/.cache/bazel/_bazel_root/e879c69dfc0f604039abdfad90e59fb0/external/io_bazel_rules_docker/container/layer_tools.bzl", line 182, in _incr_load
		zip(image["diff_id"], <1 more arguments>)
	File "/root/.cache/bazel/_bazel_root/e879c69dfc0f604039abdfad90e59fb0/external/io_bazel_rules_docker/container/layer_tools.bzl", line 182, in zip
		image["diff_id"]
key "diff_id" not found in dictionary
ERROR: Analysis of target '//build:server-images' failed; build aborted: Analysis of target '//build:push-server-images-amd64' failed
INFO: Elapsed time: 0.304s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)
make: *** [Makefile:97: images] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -97,6 +97,7 @@ images:
 	APP_VERSION=$(APP_VERSION) \
 	DOCKER_REGISTRY=$(DOCKER_REPO) \
 	bazel run \
+		--stamp \
 		--platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 \
 		//build:server-images
 

--- a/build/BUILD.bazel
+++ b/build/BUILD.bazel
@@ -49,7 +49,7 @@ multi_arch_container_push(
     name = "server-images",
     architectures = SERVER_PLATFORMS["linux"],
     docker_tags_images = {
-        "{{STABLE_DOCKER_REGISTRY}}/cert-manager-%s-{ARCH}:{{STABLE_DOCKER_TAG}}" % binary: "%s-internal" % binary
+        "{{STABLE_DOCKER_REGISTRY}}/cert-manager-%s-{ARCH}:{{STABLE_DOCKER_TAG}}" % binary: "%s.image" % binary
         for binary in DOCKERIZED_BINARIES.keys()
     },
     tags = ["manual"],

